packages/aragon-wrapper/src/cache/index.js

```diff
@@ -17,6 +17,52 @@ import * as configurationKeys from '../configuration/keys'
  *        Require the cache to downgrade to localstorage even if IndexedDB is available
  */
 export default class Cache {
+  static cacheTracker
```

Comment:
Is it possible for you to add JSDoc's anywhere here.. an example of how I was doing them: 
https://github.com/aragon/govern/blob/feature/govern-tx/packages/govern-tx/src/wallet/Wallet.ts#L36-L48